kivy/core/window/__init__.py

```diff
@@ -211,6 +211,10 @@ class WindowBase(EventDispatcher):
             Width of the window.
         `height`: int
             Height of the window.
+        `minimum_width`: int
+            Minimum width of the window
+        `minimum_height`: int
+            Minimum height of the window
```

Comment:
Added ```versionadded``` tag to both the descriptors, let me know if you need them added anywhere else.